lib/PuppeteerSharp/BrowserFetcher.cs

```diff
@@ -30,7 +30,7 @@ namespace PuppeteerSharp
     /// var browser = await await Puppeteer.LaunchAsync(new LaunchOptions { ExecutablePath = revisionInfo.ExecutablePath});
     /// </code>
     /// </example>
-    public class BrowserFetcher
+    public class BrowserFetcher : IDisposable
```

Comment:
This will mean a breaking change and a major version change.

Reply:
I was getting nervous, the more files that changed 😱